libraries/custom_appbase/tests/custom_appbase_tests.cpp

```diff
@@ -576,10 +576,11 @@ BOOST_AUTO_TEST_CASE( execute_many_from_read_only_and_read_exclusive_queues ) {
    const auto run_on_main = std::count_if(rslts.cbegin(), rslts.cend(), [&](const auto& v){ return v == app->executor().get_main_thread_id(); });
 
    BOOST_REQUIRE_EQUAL(run_on_1+run_on_2+run_on_3+run_on_main, num_expected);
-   BOOST_CHECK(run_on_1 > 0);
-   BOOST_CHECK(run_on_2 > 0);
-   BOOST_CHECK(run_on_3 > 0);
-   BOOST_CHECK(run_on_main > 0);
+   // We expect at least one to run on every thread including main, but nothing guarantees that
```

Comment:
Because we already have `BOOST_REQUIRE_EQUAL(run_on_1+run_on_2+run_on_3+run_on_main, num_expected);`, can we just remove those `BOOST_WARN` checks?

Reply:
But put a comment there.

Reply:
I'm ok with removing them. @greg7mdp what do you think?

Reply:
Sure, let's remove them.

Reply:
https://github.com/AntelopeIO/spring/pull/1530/commits/232376eded26b07d8fa595dbf01d83f79acbf4f5

Reply:
Thanks! Very clean!